python/ql/src/Expressions/UseofApply.ql

```diff
@@ -10,8 +10,10 @@
  */
 
 import python
+private import semmle.python.objects.TObject
```

Comment:
That import feels odd -- `T*` types are usually implementation details. Could we use the higher-level APIs, for example `Value::named("apply")`?

Reply:
Indeed, that should work. When the argument to `named` contains no instances of `.`, its interpretation is to look for the built-in of that name.

Reply:
Done!